tests/JWT.Tests.Core/DecodeTests.cs

```diff
@@ -67,10 +67,20 @@ public void DecodeToObject_Should_Throw_Exception_On_Invalid_Expiration_Claim()
         }
 
         [Fact]
+        public void DecodeToObject_Should_Throw_Exception_On_Null_Expiration_Claim()
```

Comment:
Sorry for confusing. Encode/Decode tests are for now-obsolete static API which we'll remove shortly. Can you please move new tests to JwtEncoder/Decoder tests?